src/widget/widget.cpp

```diff
@@ -706,7 +706,7 @@ void Widget::setStatusMessage(const QString &statusMessage)
 void Widget::reloadHistory()
 {
     for (auto f : FriendList::getAllFriends())
-        f->getChatForm()->loadHistory(QDateTime::currentDateTime().addDays(-7), true);
+        f->getChatForm()->loadHistory(QDateTime::currentDateTime().addDays(-1), true);
```

Comment:
This change is NOT mentioned in the PR description or any of the commit messages.

Changing a `0` into a `this` to center some message boxes without mentioning it just annoys me a bit, but this is going too far.
Please don't sneak in unrelated changes with pull requests, or at least mention it in the description.
So apparently the only thing this does is that when the password is changed, we would reload 1 day of history instead of 1 week. This doesn't make much sense since in `Friend::loadHistory` we still load 1 week.

What were you trying to do?